main.go

```diff
@@ -129,25 +129,25 @@ func drawCard(done chan bool, cards []deck.Card, ticker *time.Ticker) {
 }
 
 // scoring function
-func scoring(snap bool) {
+func scoring(snap bool) int{
 
 	if snap {
 		if presentCards[0].Rank == presentCards[1].Rank {
 			score++
 			fmt.Println("\nYour score is:", score)
-			return
+			return score
```

Comment:
It's a little bit confusing to return the global variable `score` here. Usually if we return something we want to try and return something that is a limited to the scope of the function as possible.

For example, we might think about `scoring` returning the change in score that needs to be made (i,e +1/0/-1) and applying that to the global game score where we call `scoring` instead.